### PR TITLE
feat: create a input number component

### PR DESCRIPTION
**Pull request by liordino**
feat: created a number input component (base and formik)

fix: replace numeric TextInput fields on the Cashout, Commissions,
Cashboxes, Notifications, Operator Info and Terms & Conditions pages

### CODE REVIEW

**Comment by liordino** [comment]
[Create a input number component](https://trello.com/c/yCWQjz7m/174-create-a-input-number-component)

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 1]
```diff
@@ -54,35 +54,47 @@ const getOverridesFields = (getData, currency) => {
       name: 'cashIn',
       display: 'Cash-in',
       width: 140,
-      input: TextInput,
+      input: NumberInput,
       textAlign: 'right',
-      suffix: '%'
+      suffix: '%',
+      inputProps: {
+        decimalPlaces: 0
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/helper.js]: We should allow three decimal places here.

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 2]
```diff
@@ -54,35 +54,47 @@ const getOverridesFields = (getData, currency) => {
...
       name: 'cashOut',
       display: 'Cash-out',
       width: 140,
-      input: TextInput,
+      input: NumberInput,
       textAlign: 'right',
-      suffix: '%'
+      suffix: '%',
+      inputProps: {
+        decimalPlaces: 0
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/helper.js]: three decimal places here as well

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 3]
```diff
@@ -93,16 +105,22 @@ const mainFields = currency => [
     display: 'Cash-in',
     width: 169,
     size: 'lg',
-    input: TextInput,
-    suffix: '%'
+    input: NumberInput,
+    suffix: '%',
+    inputProps: {
+      decimalPlaces: 0
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/helper.js]: same as above

File: new-lamassu-admin/src/pages/Commissions/helper.js [hunk 4]
```diff
@@ -93,16 +105,22 @@ const mainFields = currency => [
...
     name: 'cashOut',
     display: 'Cash-out',
     width: 169,
     size: 'lg',
-    input: TextInput,
-    suffix: '%'
+    input: NumberInput,
+    suffix: '%',
+    inputProps: {
+      decimalPlaces: 0
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/helper.js]: same as above

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**